spec/specialist_publisher_spec.rb

```diff
@@ -143,4 +143,36 @@
       expect_unpublished_document
     end
   end
+
+  feature "Documents with attachment" do
+    let(:title) { Faker::Book.author }
+    let(:summary) { "Documents with attachment" }
+    let(:file) { File.expand_path("./fixtures/my-file-for-upload.pdf", File.dirname(__FILE__)) }
```

Comment:
Might be nicer to name this file something a bit more related to the content of it, since it'd be a confusing name once there was a second type of file for testing uploads?